producer.py

```diff
@@ -15,18 +15,23 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-"""Check for new releases on Python index and mark new packages in the graph database."""
+"""Check for new releases on Python index and mark new packages in the graph database.
+
+Produces messages regarding new releases for packages registered in the graph database.
+"""
 
 import sys
 import logging
 import typing
 
+from faust import cli
+
 import os
 import requests
 import yaml
+import asyncio
 
-import click
-from prometheus_client import CollectorRegistry, Gauge, push_to_gateway
+from prometheus_client import CollectorRegistry, Gauge
```

Comment:
I think we can remove all metrics associated with the producer, if the PushGateway is not used there is no way to view these metrics.

Reply:
thanks, https://github.com/thoth-station/package-releases-job/pull/449/commits/20ef1ccb522b8fa3dbb273c81fea357ae35e61bf